### DISCUSSION TITLE
茉莉花插件知网助手只有一个且无法使用

### DISCUSSION THREAD

**Discussion by Knight-YZ** [l0o0/jasminum#329] [Q&A]
<img width="323" alt="屏幕截图 2024-10-29 102244" src="https://github.com/user-attachments/assets/77b343e7-d2a5-4646-9cb2-2db12b0fc144">
<img width="408" alt="image" src="https://github.com/user-attachments/assets/753cb92b-1745-4497-af04-549790203fd5">

**Comment by Maytree42** [comment]
我也发现类似情况，来补充一下……目前使用7.0.11 (64-bit)Zotero，1.0.0-20版本插件，此外还安装了5.3.5版本Ethereal Style，2.0.13版本Translate for Zotero。打开Zotero后立刻对条目点击右键，可以看到知网助手有3个功能，但下一秒再点右键，就只剩一个功能了。
![屏幕截图 2024-12-14 175437](https://github.com/user-attachments/assets/e9a13624-d734-4887-bd53-f841e74b48ce)
![屏幕截图 2024-12-14 175312](https://github.com/user-attachments/assets/a40c13f8-7a87-4dee-b81b-fa75ff39989f)
这个在Zotero6的时候就有了……停用其他插件问题仍存在。希望开发者能看到留言……

**Reply by jiaojiaodubai** [reply]
请提供视野更大的截图

**Reply by Maytree42** [reply]
> 请提供视野更大的截图

![屏幕截图 2024-12-25 212058](https://github.com/user-attachments/assets/c4ebf14b-45ba-4777-b382-d152443b2edf)
